test/data/ipfs-client-updater/ipfs-client-updater-linux.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Just double-checking: the PEM files used for tests should be unique and not used in any production code. Can you confirm? Thanks!

Reply:
the private key I used is a copy of the private key used in the Tor test implementation at https://github.com/brave/brave-core/blob/master/test/data/tor-client-updater/tor-client-updater-mac.pem